packages/main/src/requests/request.ts

```diff
@@ -62,17 +62,21 @@ function getClientHeaders(): string {
 export async function makeRequest(
   url: RequestUrl,
   body: string,
+  requestInit?: RequestInit,
 ): Promise<Response> {
   let response;
   try {
     response = await fetch(url.toString(), {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-        "x-goog-api-client": getClientHeaders(),
-        "x-goog-api-key": url.apiKey,
+      ...requestInit,
+      ...{
```

Comment:
Why are the default properties put into an object and re-spread?

Reply:
it was a mistake, commit 748ac95c6ee20c7b234632f81e74cefc09d0d039 solve this